BareMetal.tf

```diff
@@ -7,6 +7,32 @@ provider "metal" {
   auth_token = var.metal_auth_token
 }
 
+locals {
+  ssh_key_name = "metal-key"
```

Comment:
This needs to be updated as well to match the style of the Anthos/Tanzu-style keypair generator like we re-used here:

```hcl
ssh_key_name        = format("openstack-%s", random_string.cloud.b64_url)
```